vignettes/safestats-vignette.Rmd

```diff
@@ -252,16 +246,12 @@ plot(simResultDeltaTrueLargerThanDeltaMin)
 Hence, this means that if the true effect is larger than what was planned for, the safe test will detect this larger effect earlier on, which results in a further increase of efficiency. 
 
 ### Optional stopping does not causes safe tests to overreject the null, but is problematic for $p$-values
-The previous examples highlight how optional stopping results in an increase in power, i.e., the chance of rejecting the null is increased, when the alternative is true. When the null holds true, however, the rejection rate should be low, at least not larger than the tolerable type I error rate. Here we show that optional stopping results in the type I error rate of the safe test to *not* exceed $\alpha$, whereas early stopping with classical $p$-value tests *does* result in the exceedance of the prescribed $\alpha$-level. In other words, optional stopping with $p$-values leads to an increased risk of falsely claiming that a medicine is effective, while in reality the effect is abscent. 
+The previous examples highlight how optional stopping results in an increase in power, i.e., the chance of rejecting the null is increased, when the alternative is true. When the null holds true, however, the rejection rate should be low, at least not larger than the tolerable type I error rate. Here we show that optional stopping results in the type I error rate of the safe test to *not* exceed $\alpha$, whereas early stopping with classical $p$-value tests *does* result in the exceedance of the prescribed $\alpha$-level. In other words, optional stopping with $p$-values leads to an increased risk of falsely claiming that a medicine is effective, while in reality the effect is absent. 
 
 For this purpose we run the code
 
 ```{r}
-# alpha <- 0.05
```

Comment:
The alpha, beta and deltaMin are defined above, but I wanted to keep them in the code, so people know what their value is. Perhaps my intention wasn't clear and we should add something such as

    # Recall:
    # alpha <- 0.05
    # beta <- 0.2
    # deltaMin <- 9/(sqrt(2)*15)      # = 0.42